RabbitMQ.Stream.Client/PublicAPI.Unshipped.txt

```diff
@@ -7,6 +7,7 @@ RabbitMQ.Stream.Client.AuthMechanism.External = 1 -> RabbitMQ.Stream.Client.Auth
 RabbitMQ.Stream.Client.AuthMechanism.Plain = 0 -> RabbitMQ.Stream.Client.AuthMechanism
 RabbitMQ.Stream.Client.AuthMechanismNotSupportedException
 RabbitMQ.Stream.Client.AuthMechanismNotSupportedException.AuthMechanismNotSupportedException(string s) -> void
+RabbitMQ.Stream.Client.Chunk.Crc.get -> uint
```

Comment:
Unfortunately this is a breaking change in the public API, because a library user who upgrades will get this when they compile:

```
error CS0266: Cannot implicitly convert type 'int' to 'uint'. An explicit conversion exists (are you missing a cast?)
```

...and I'm sure some sort of nasty exception at run time.

Is this change necessary?

Reply:
I had to change `  public readonly struct Deliver : ICommand` struct since the CRC coming from the server is a `uint` and not an `int`. 

Users don't use this class directly in fact I think the class `Deliver` as many others should per internal/private.